Simplenote/src/main/java/com/automattic/simplenote/utils/MatchOffsetHighlighter.java

```diff
@@ -87,6 +89,24 @@ public static void highlightMatches(Spannable content, String matches, int colum
         }
     }
 
+    // Returns the character location of the first match (3rd index)
```

Comment:
Why is the first match the third index?

Reply:
I can see how that'd be confusing. There's 4 digits in a space-separated string for a `match`. The 3rd digit is the one that gets us the start of the match string in the text. I added a comment that _hopefully_ clears that up!